documentation/docs.html

```diff
@@ -91,6 +90,55 @@ <h2 class="headline-2 text-sub-head">Text avatars</h2>
                         </div>
                     </div>
                 </section>
+                <section id="alerts-sec">
+                    <div class="comp-intro">
+                        <h1 class="headline-1 text-head">Alerts</h1>
+                        <p class="headline-3">
+                            Alerts are used to attract user's attention for important information without interrupting
+                            the user's flow.
+                        </p>
+                    </div>
+                    <div class="comp-details">
+                        <h2 class="headline-2 text-sub-head">Alerts without close button</h2>
+                        <p class="text-mid">
+                            You can use <span>alerts</span> class to create an alert. Use <span>alert-pri, alert-suc,
+                                alert-war, alert-dan</span> for diffrent types of alerts.
+                            You can use diffrent style classes for diffrent stylling.
+                        </p>
+                        <div class="comp-examples center-flex flex-d-col">
+                            <div class="alerts alert-pri padd-1r marg-1r bor-small text-mid">
+                                <p class="alert">
+                                    This is a Primary alert.
+                                </p>
```

Comment:
Where is the "class" that you have used in the p tag, or is it redundant.

Reply:
Ohh, It's just redundant. I forgot to remove it.

Reply:
Ok, rest is fine.